src/Schema/AbstractSchemaManager.php

```diff
@@ -812,7 +813,11 @@ protected function _getPortableTableColumnList(string $table, string $database,
     {
         $list = [];
         foreach ($rows as $row) {
-            $column = $this->_getPortableTableColumnDefinition($row);
+            try {
+                $column = $this->_getPortableTableColumnDefinition($row);
+            } catch (UnknownColumnType $unknownTypeException) {
+                throw UnknownColumnType::newWithContext($unknownTypeException->getRequestedType(), $table);
```

Comment:
`$unknownTypeException` and its stack trace are destroyed here. Please always attach the an exception as `$previous` when upcasting it.

Reply:
Good catch, fixed it.